### PR TITLE
Leverage formattable strings to generate SQL and parameters.

### PR DESCRIPTION
**Pull request by ejball**
Inspired by #20.

Could be extended over time to support differences among SQL syntaxes, escaping SQL keywords, generating names and parameters from DTOs for inserts, injecting literals instead of parameters, joining SQL fragments, etc.

### CODE REVIEW

File: src/Faithlife.Data/DbConnector.cs [hunk 1]
```diff
@@ -132,6 +133,54 @@ public abstract class DbConnector : IDisposable, IAsyncDisposable
...
+		/// </summary>
+		/// <param name="sql">The parameterized SQL.</param>
+		/// <param name="parameters">Additional command parameters.</param>
+		public DbConnectorCommand Command(Sql sql, params (string Name, object? Value)[] parameters) => Command(sql, DbParameters.Create(parameters));
+
+		/// <summary>
+		/// Creates a new command.
+		/// </summary>
+		/// <param name="sql">The parameterized SQL.</param>
+		public DbConnectorCommand CommandFormat(FormattableString sql) => Command(Sql.Format(sql));
```
Comment (bgrainger) [inline src/Faithlife.Data/DbConnector.cs:168]: Do these overloads add enough utility to justify the risks around `FormattableString` (i.e., it downgrades to `string`--with injection--too easily)?

Comment (ejball) [reply src/Faithlife.Data/DbConnector.cs:168]: Not necessarily. [Comparison here.](https://github.com/Faithlife/FaithlifeData/pull/27/files#diff-961e985c94b8fef1d88f367c09d463d48ec23251a23a4eb5cd72efc78b51e304) I'm willing to drop the `CommandFormat` overloads. If someone really wants them they can always write their own extension methods.

Comment (bgrainger) [reply src/Faithlife.Data/DbConnector.cs]: Yes, this doesn't seem to add a lot of value to Faithlife.Data itself; I don't see the need to provide two very similar ways to do the same thing.

**Review by ddunkin — COMMENTED** [review]
I'd still rather parameters not require a `param` format specifier, but be the default behavior. I don't think the format specifier increases readability or reduces the chances of an error enough to be worth requiring the extra typing and noise. Having parameter bet the default makes an actual potential injection vulnerability via `raw` even more obvious.

**Comment by ejball** [comment]
Two against one; I surrender. I still maintain that it looks too much like simple injection, but concede that the extra typing and noise may not be worth it. [`6b724c5` (#27)](https://github.com/Faithlife/FaithlifeData/pull/27/commits/6b724c5b45f487fc0b9c06c074ca1436ee7c4a15)

**Review by ddunkin — APPROVED** [review]
I can contribute a docs change if you'd like.

**Comment by ejball** [comment]
> I can contribute a docs change if you'd like.

That would be great!

**Pull request merged**